sequelizeDatabase/sequelFunctions.js

```diff
@@ -264,54 +265,78 @@ insertFromCsvLine = (
     Comments,
     "\n"
   );
-  insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
-    .then(newPerson => {
-      insertCompany(PortfolioCompanyName, FundID).then(originalCompany => {
-        insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion)
-          .then(() => {
-            //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
-            insertEmployeeHistory(
-              UserID,
-              newPerson.IndividualID,
-              originalCompany.CompanyID,
-              OriginalPostion,
-              OriginalStartDate,
-              OriginalEndDate
-            ).then(() => {
-              insertCompany(CurrentEmployer, FundID).then(newCompany => {
-                insertEmployeeHistory(
-                  UserID,
-                  newPerson.IndividualID,
-                  newCompany.CompanyID,
-                  CurrentPostion,
-                  OriginalStartDate,
-                  OriginalEndDate
-                );
+  //returns IndividualID for use in processing quarter updates
+  return new Promise((resolve, reject) => {
```

Comment:
Returning a promise so that the data can be used in backend to call for quarter updates.